src/DependencyInjection/Configuration.php

```diff
@@ -231,24 +231,16 @@ public function getConfigTreeBuilder()
                                             ->ifArray()
                                             ->then(static function ($items) {
                                                 foreach ($items as $key => $item) {
-                                                    if (\is_array($item)) {
-                                                        if (!\array_key_exists('label', $item) || !\array_key_exists('route', $item)) {
```

Comment:
I think the only thing we need to add is the change from null to empty string here, instead of removing this code